anda/lib/libhelium/libhelium.spec

```diff
@@ -22,8 +22,8 @@ BuildRequires:  pkgconfig(gtk4) >= 4.4
 Requires: gtk4 >= 4.4
 Requires: glib2 >= 2.66.0
 Requires: libgee >= 0.20
-Requires: tau-helium >= 1.1.25
-Requires: tau-hydrogen
+Requires: helium-gtk-theme >= 1.1.25
```

Comment:
maybe remove the version constraint here IMO

Reply:
that's just the way it was before